Components/Avatar/avatar.html

```diff
@@ -0,0 +1,161 @@
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="../../css/navbar.css">
+	<link rel="stylesheet" href="../../css/sidebar.css">
+	<link rel="stylesheet" href="../../css/docs-page.css">
+	<link rel="stylesheet" href="../../css/boilerplate.css">
+	<link rel="stylesheet" href="../Avatar/avatar.css">
+	<title>Avatar - moment ui</title>
+</head>
+
+<body>
+	<div class="container">
+		<header class="nav">
+			<div class="nav-left">
+				<span class="material-icons sidebar-menu-icon">menu</span>
+				<a href="/index.html">
+					<span class="brand">
+						<span class="brand-title">moment ui</span>
+					</span>
+				</a>
+			</div>
+			<div class="nav-right">
+				<a href="#"><span class="nav-link">Docs</span></a>
+				<a href="#">
+					<img src="../../assets/github-icon.svg" class="nav-icon"></img>
+				</a>
+			</div>
+		</header>
+		<div class="doc-page-container">
+			<aside class="sidebar" id="sidebar">
+				<div class="">
+					<span class="sidebar-close-icon material-icons">close</span>
+					<h4 class="sidebar-category">Get Started</h4>
+					<ul class="sidebar-subcategory" role="list">
+						<li>
+							<a id="sidebar-links" href="/get started/quick-start.html">Quick Start</a>
+						</li>
+						<li>
+							<a id="sidebar-links" href="/get started/colors.html">Colors</a>
+						</li>
+						<li>
+							<a id="sidebar-links" href="/get started/typography.html">Typography</a>
+						</li>
+					</ul>
+				</div>
+				<div class="">
+					<h4 class="sidebar-category">Components</h4>
+					<ul class="sidebar-subcategory" role="list">
+						<li>
+							<a id="sidebar-links" href="/components/Alert/alert.html">Alert</a>
+						</li>
+						<li>
+							<a class="active" id="sidebar-links" href="/components/Avatar/avatar.html">Avatar</a>
+						</li>
+						<li>
+							<a id="sidebar-links" href="/components/Badge/badge.html">Badge</a>
+						</li>
+					</ul>
+				</div>
+			</aside>
+			<div class="doc-content">
+				<h1 class="content-title">Avatar</h1>
+				<p class="content-description">Avatars are used to give a recognisable visual identity such as profile
+					pictures, usually found in circular shapes.
+				</p>
+				<div class="divider"></div>
+				<!-- Standard Avatars  -->
+				<h2 class="content-category">Standard Avatars</h2>
+				<div class="example-container avatar-preview">
+					<img class="avatar" src="/assets/avatar-img1.jpeg" alt="gentleman-image" />
+					<img class="avatar" src="/assets/avatar-img2.jpeg" alt="gentleman-image" />
+					<img class="avatar" src="/assets/avatar-img4.jpeg" alt="lady-image" />
+					<img class="avatar" src="/assets/avatar-img3.jpeg" alt="cartoon-image" />
+				</div>
+				<div class="code-snippet">
+					<iframe src="https://carbon.now.sh/embed/XxwRDKhDOnd6pWCuF8Mo"
+						style="width: 100%; height: 159px; border:0; transform: scale(1); overflow:hidden;"
+						sandbox="allow-scripts allow-same-origin">
+					</iframe>
```

Comment:
You could move the iframe styling to one of the common style sheets to keep the html clean and avoid in-line styling in general.

Reply:
Okay. I'll refactor it. Thank you.